taxonomies/nutrients.txt

```diff
@@ -1469,7 +1470,7 @@ nb:fett
 nl:vet
 nn:feitt
 pl:tłuszcz
-pt:gordura
+pt:Gorduras/lípidos, Gordura, Lípidos
```

Comment:
```suggestion
pt:Gorduras, Gordura, Lípidos
```

Reply:
These suggestions by me where to solve a minor/big problem: OpenFoodFacts at this moment only have 1 Portuguese interface translation. European Portuguese and Brazilian Portuguese are not very different, but sometimes the words are not the same. In Brazil "Fats" are shown as "Gorduras" in nutrition labels and in Portugal "Lípidos" (don't know in other Portuguese speaking countries, generally they use European Portuguese). So translating "Fats" as "Gorduras/lípidos" solves this problem.

I'm guessing this translation is shown in OpenFoodFacts while editing nutrition facts in a product, right? And only the first word "Gorduras" in "pt:Gorduras, Gordura, Lípidos" is shown, the others "Gordura, Lípidos" are shown if we search/add another nutrient. I'm I wrong?

Reply:
you are right @laralem

Reply:
But I'm not sure that's a good way to deal with it. @stephanegigandet what is your opinion ?

Reply:
sounds good to me until we have a better way to deal with language variants